python/tk_desktop/setup_project.py

```diff
@@ -46,11 +48,18 @@ def do_setup(self):
             if is_on_top: 
                 self._set_top_window_on_top(False)
 
-
             setup = adminui.SetupProjectWizard(self.project, self)
             ret = setup.exec_()
             self.setup_finished.emit(ret == setup.Accepted)
 
+        except TankErrorProjectIsSetup, e:
+            error_dialog = ErrorDialog("Toolkit Setup Error",
```

Comment:
A bit unfair to ask this during your code review since you weren't the one who introduced the limitation, but why don't we force the project creation like we do with tank?